{
  "uid" : "8c47f145c7396c12",
  "name" : "test 04.02.07 create an account with incorrect confirm password",
  "fullName" : "tests.test_create_an_account.TestCreateAnAccount#test_tc_04_02_07_create_account_with_incorrect_confirm_password",
  "historyId" : "09679f850e6cf760d69d28578925e32f",
  "time" : {
    "start" : 1735438075879,
    "stop" : 1735438081010,
    "duration" : 5131
  },
  "description" : " Verify that customer can't Create An Account with incorrect confirm password",
  "descriptionHtml" : "<p>Verify that customer can't Create An Account with incorrect confirm password</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564ba9c6931a <unknown>\n#1 0x564ba977f6e0 <unknown>\n#2 0x564ba97ce3e6 <unknown>\n#3 0x564ba97ce681 <unknown>\n#4 0x564ba9813b04 <unknown>\n#5 0x564ba97f248d <unknown>\n#6 0x564ba9810ed7 <unknown>\n#7 0x564ba97f2203 <unknown>\n#8 0x564ba97c0cc0 <unknown>\n#9 0x564ba97c1c9e <unknown>\n#10 0x564ba9c36d0b <unknown>\n#11 0x564ba9c3ac92 <unknown>\n#12 0x564ba9c23b3c <unknown>\n#13 0x564ba9c3b807 <unknown>\n#14 0x564ba9c090df <unknown>\n#15 0x564ba9c58578 <unknown>\n#16 0x564ba9c58740 <unknown>\n#17 0x564ba9c68196 <unknown>\n#18 0x7f9a7da9ca94 <unknown>\n#19 0x7f9a7db29c3c <unknown>",
  "statusTrace" : "self = <test_create_an_account.TestCreateAnAccount object at 0x7fcc684b2050>\ncreate_account_page = <pages.create_an_account_page.CreateAccountPage object at 0x7fcc6905b490>\n\n    @allure.title('test 04.02.07 create an account with incorrect confirm password')\n    def test_tc_04_02_07_create_account_with_incorrect_confirm_password(self, create_account_page):\n        \"\"\" Verify that customer can't Create An Account with incorrect confirm password\"\"\"\n>       message = create_account_page.create_with_incorrect_confirm_password()\n\ntests/test_create_an_account.py:208: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/create_an_account_page.py:365: in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e33e9a543e4bb78cc1525f8c2454971c\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fcc674918a0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564ba9c6931a <unknown>\nE       #1 0x564ba977f6e0 <unknown>\nE       #2 0x564ba97ce3e6 <unknown>\nE       #3 0x564ba97ce681 <unknown>\nE       #4 0x564ba9813b04 <unknown>\nE       #5 0x564ba97f248d <unknown>\nE       #6 0x564ba9810ed7 <unknown>\nE       #7 0x564ba97f2203 <unknown>\nE       #8 0x564ba97c0cc0 <unknown>\nE       #9 0x564ba97c1c9e <unknown>\nE       #10 0x564ba9c36d0b <unknown>\nE       #11 0x564ba9c3ac92 <unknown>\nE       #12 0x564ba9c23b3c <unknown>\nE       #13 0x564ba9c3b807 <unknown>\nE       #14 0x564ba9c090df <unknown>\nE       #15 0x564ba9c58578 <unknown>\nE       #16 0x564ba9c58740 <unknown>\nE       #17 0x564ba9c68196 <unknown>\nE       #18 0x7f9a7da9ca94 <unknown>\nE       #19 0x7f9a7db29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1735436375854,
      "stop" : 1735436375914,
      "duration" : 60
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1735438074613,
      "stop" : 1735438075787,
      "duration" : 1174
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1735436375915,
      "stop" : 1735436375915,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "create_account_page",
    "time" : {
      "start" : 1735438075787,
      "stop" : 1735438075878,
      "duration" : 91
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1735438075787,
        "stop" : 1735438075878,
        "duration" : 91
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "testStage" : {
    "description" : " Verify that customer can't Create An Account with incorrect confirm password",
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564ba9c6931a <unknown>\n#1 0x564ba977f6e0 <unknown>\n#2 0x564ba97ce3e6 <unknown>\n#3 0x564ba97ce681 <unknown>\n#4 0x564ba9813b04 <unknown>\n#5 0x564ba97f248d <unknown>\n#6 0x564ba9810ed7 <unknown>\n#7 0x564ba97f2203 <unknown>\n#8 0x564ba97c0cc0 <unknown>\n#9 0x564ba97c1c9e <unknown>\n#10 0x564ba9c36d0b <unknown>\n#11 0x564ba9c3ac92 <unknown>\n#12 0x564ba9c23b3c <unknown>\n#13 0x564ba9c3b807 <unknown>\n#14 0x564ba9c090df <unknown>\n#15 0x564ba9c58578 <unknown>\n#16 0x564ba9c58740 <unknown>\n#17 0x564ba9c68196 <unknown>\n#18 0x7f9a7da9ca94 <unknown>\n#19 0x7f9a7db29c3c <unknown>",
    "statusTrace" : "self = <test_create_an_account.TestCreateAnAccount object at 0x7fcc684b2050>\ncreate_account_page = <pages.create_an_account_page.CreateAccountPage object at 0x7fcc6905b490>\n\n    @allure.title('test 04.02.07 create an account with incorrect confirm password')\n    def test_tc_04_02_07_create_account_with_incorrect_confirm_password(self, create_account_page):\n        \"\"\" Verify that customer can't Create An Account with incorrect confirm password\"\"\"\n>       message = create_account_page.create_with_incorrect_confirm_password()\n\ntests/test_create_an_account.py:208: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/create_an_account_page.py:365: in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e33e9a543e4bb78cc1525f8c2454971c\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fcc674918a0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564ba9c6931a <unknown>\nE       #1 0x564ba977f6e0 <unknown>\nE       #2 0x564ba97ce3e6 <unknown>\nE       #3 0x564ba97ce681 <unknown>\nE       #4 0x564ba9813b04 <unknown>\nE       #5 0x564ba97f248d <unknown>\nE       #6 0x564ba9810ed7 <unknown>\nE       #7 0x564ba97f2203 <unknown>\nE       #8 0x564ba97c0cc0 <unknown>\nE       #9 0x564ba97c1c9e <unknown>\nE       #10 0x564ba9c36d0b <unknown>\nE       #11 0x564ba9c3ac92 <unknown>\nE       #12 0x564ba9c23b3c <unknown>\nE       #13 0x564ba9c3b807 <unknown>\nE       #14 0x564ba9c090df <unknown>\nE       #15 0x564ba9c58578 <unknown>\nE       #16 0x564ba9c58740 <unknown>\nE       #17 0x564ba9c68196 <unknown>\nE       #18 0x7f9a7da9ca94 <unknown>\nE       #19 0x7f9a7db29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Create an account with incorrect confirm password",
      "time" : {
        "start" : 1735438075879,
        "stop" : 1735438081009,
        "duration" : 5130
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564ba9c6931a <unknown>\n#1 0x564ba977f6e0 <unknown>\n#2 0x564ba97ce3e6 <unknown>\n#3 0x564ba97ce681 <unknown>\n#4 0x564ba9813b04 <unknown>\n#5 0x564ba97f248d <unknown>\n#6 0x564ba9810ed7 <unknown>\n#7 0x564ba97f2203 <unknown>\n#8 0x564ba97c0cc0 <unknown>\n#9 0x564ba97c1c9e <unknown>\n#10 0x564ba9c36d0b <unknown>\n#11 0x564ba9c3ac92 <unknown>\n#12 0x564ba9c23b3c <unknown>\n#13 0x564ba9c3b807 <unknown>\n#14 0x564ba9c090df <unknown>\n#15 0x564ba9c58578 <unknown>\n#16 0x564ba9c58740 <unknown>\n#17 0x564ba9c68196 <unknown>\n#18 0x7f9a7da9ca94 <unknown>\n#19 0x7f9a7db29c3c <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/create_an_account_page.py\", line 365, in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Find a visible element",
        "time" : {
          "start" : 1735438075879,
          "stop" : 1735438081009,
          "duration" : 5130
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564ba9c6931a <unknown>\n#1 0x564ba977f6e0 <unknown>\n#2 0x564ba97ce3e6 <unknown>\n#3 0x564ba97ce681 <unknown>\n#4 0x564ba9813b04 <unknown>\n#5 0x564ba97f248d <unknown>\n#6 0x564ba9810ed7 <unknown>\n#7 0x564ba97f2203 <unknown>\n#8 0x564ba97c0cc0 <unknown>\n#9 0x564ba97c1c9e <unknown>\n#10 0x564ba9c36d0b <unknown>\n#11 0x564ba9c3ac92 <unknown>\n#12 0x564ba9c23b3c <unknown>\n#13 0x564ba9c3b807 <unknown>\n#14 0x564ba9c090df <unknown>\n#15 0x564ba9c58578 <unknown>\n#16 0x564ba9c58740 <unknown>\n#17 0x564ba9c68196 <unknown>\n#18 0x7f9a7da9ca94 <unknown>\n#19 0x7f9a7db29c3c <unknown>\n\n",
        "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ {
          "name" : "Find a present element",
          "time" : {
            "start" : 1735438075879,
            "stop" : 1735438081009,
            "duration" : 5130
          },
          "status" : "broken",
          "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564ba9c6931a <unknown>\n#1 0x564ba977f6e0 <unknown>\n#2 0x564ba97ce3e6 <unknown>\n#3 0x564ba97ce681 <unknown>\n#4 0x564ba9813b04 <unknown>\n#5 0x564ba97f248d <unknown>\n#6 0x564ba9810ed7 <unknown>\n#7 0x564ba97f2203 <unknown>\n#8 0x564ba97c0cc0 <unknown>\n#9 0x564ba97c1c9e <unknown>\n#10 0x564ba9c36d0b <unknown>\n#11 0x564ba9c3ac92 <unknown>\n#12 0x564ba9c23b3c <unknown>\n#13 0x564ba9c3b807 <unknown>\n#14 0x564ba9c090df <unknown>\n#15 0x564ba9c58578 <unknown>\n#16 0x564ba9c58740 <unknown>\n#17 0x564ba9c68196 <unknown>\n#18 0x7f9a7da9ca94 <unknown>\n#19 0x7f9a7db29c3c <unknown>\n\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "locator",
            "value" : "('id', 'firstname')"
          }, {
            "name" : "timeout",
            "value" : "5"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "locator",
          "value" : "('id', 'firstname')"
        }, {
          "name" : "timeout",
          "value" : "10"
        } ],
        "stepsCount" : 1,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 2,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "35b29a0bf6c2dc35",
      "name" : "Screenshot",
      "source" : "35b29a0bf6c2dc35.png",
      "type" : "image/png",
      "size" : 87712
    }, {
      "uid" : "c4563d7e11ec0cf6",
      "name" : "HTML source",
      "source" : "c4563d7e11ec0cf6.html",
      "type" : "text/html",
      "size" : 6658
    } ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1735438081119,
      "stop" : 1735438081183,
      "duration" : 64
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1735439409300,
      "stop" : 1735439409300,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Create An Account"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_create_an_account"
  }, {
    "name" : "subSuite",
    "value" : "TestCreateAnAccount"
  }, {
    "name" : "host",
    "value" : "fv-az1110-580"
  }, {
    "name" : "thread",
    "value" : "2884-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_create_an_account"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "8c47f145c7396c12.json",
  "parameterValues" : [ ]
}